### PR TITLE
Remove the exporting of the io.swagger.annotations package

### PR DESCRIPTION
**Pull request by geethkokila**

### CODE REVIEW

File: swagger-codegen/2.2.0.wso2v3/pom.xml [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thusithathilina) [inline swagger-codegen/2.2.0.wso2v3/pom.xml]: licence header should be 2017

File: swagger-codegen/2.2.0.wso2v3/pom.xml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                            io.swagger.models; version="[1.5.8,1.6)"
+                        </Import-Package>
+                        <Private-Package>
+                        </Private-Package>
+                        <Include-Resource>
+                            {maven-resources}
+                        </Include-Resource>
+                        <Embed-Dependency>
+                        </Embed-Dependency>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```
Comment (thusithathilina) [inline swagger-codegen/2.2.0.wso2v3/pom.xml]: Do we need Dynamic Import?

File: swagger-codegen/2.2.0.wso2v3/pom.xml (hunk 1 above)
Comment (geethkokila) [inline swagger-codegen/2.2.0.wso2v3/pom.xml]: Fixed in https://github.com/wso2/orbit/pull/265/commits/84f79dc486c94adbc41108d6c7300d8553a3f746

File: swagger-codegen/2.2.0.wso2v3/pom.xml (hunk 2 above)
Comment (thusithathilina) [inline swagger-codegen/2.2.0.wso2v3/pom.xml]: This is require for code generation part

**Pull request merged**